shared/util/feature-flags.js.flow

```diff
@@ -4,7 +4,6 @@ export type FeatureFlags = {|
   airdrop: boolean,
   chatIndexProfilingEnabled: boolean,
   dbCleanEnabled: boolean,
-  enableDeleteFolder: boolean,
```

Comment:
I think design would need a QA pass before we can release this, and likely we'll need to tweak the modal popup. If you get rid of this flag in this PR, could you follow-up with design and make sure we either get it right before next release?